android-sdk/src/main/java/org/sagebionetworks/bridge/android/BridgeConfig.java

```diff
@@ -98,6 +98,11 @@ public int getAppVersion() {
     }
 
     @NonNull
+    public String getAppVersionName() {
+        return applicationContext.getResources().getString(R.string.osb_app_version_name);
```

Comment:
It used to be in the form "version 1.0.8, build 27", where the version was what matched the app store version and the build number was what matched the user_agent header. However, we've stopped enforcing (or even parsing) this. I'd still recommend keeping that format (or one similar to it), since it includes both the researcher-facing version number and the dev-facing version number.

(If you're wondering why there's more than one version number, that's a good question. I assume it comes from the days of iOS development, and I'm not sure it was "designed" as much as it "came into existence".)